src/components/StandardFacets.tsx

```diff
@@ -61,7 +68,7 @@ export function StandardFacets(props: StandardFacetsProps) {
               <FilterGroup
                 fieldId={f.fieldId}
                 filterOptions={f.options.map(o => {
-                  return { ...o, resultsCount: o.count };
+                  return showOptionCounts ? { ...o, resultsCount: o.count } : { ...o };
```

Comment:
can we just return `o` if it's false? no need to perform spread assignment (or maybe do mapping only if showOptionCounts is true)

Reply:
True!  I changed it to return `o` on false